backend/app/routes/scraping.py

```
@@ -246,6 +247,44 @@ async def quick_scrape_infozagreb(
+        async def run_visit_task():
+            return await scrape_visitopatija_events(max_pages=request.max_pages)
+
+        background_tasks.add_task(run_visit_task)
+
+        return ScrapeResponse(
+            status="accepted",
+            message=f"VisitOpatija.com scraping task started in background for {request.max_pages} pages",
+            task_id=task_id,
+        )
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Failed to start VisitOpatija scraping: {str(e)}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)